semgrep-core/src/analyzing/AST_to_IL.ml

```diff
@@ -126,6 +126,12 @@ let var_of_id_info id id_info =
   in
   { ident = id; sid; id_info }
 
+let var_of_name name =
+  match name with
+  | G.Id (id, id_info) -> var_of_id_info id id_info
+  | G.IdQualified { G.name_last = id, _typeargsTODO; name_info = id_info; _ } ->
```

Comment:
FYI this type changed quite a lot with the introduction of C++. There is now a name_middle, and name_top
field.